### PR TITLE
UI fixes

### PR DESCRIPTION
**Pull request by Gauravjeetsingh**
Fixes the issues mentioned in #1837 except the next line functionality. I will open a separate PR for next line.

- [x] 50px v 65px line height - increase line height by 30%
- [x] Make font bold
- [x] Line hover background (light grey)
- [x] Spacing of buttons at the top of navigator
- [x] New icons inc. hide screen
- [x] Hide icon for lines.
- [x] Move “make home” button to right and show on line hover.
- [ ] A way to show what line would be shown next if space bar pressed.
- [x] Fixing the scroll
- [x] Narrow toolbar
- [ ] Change Icon for ceremonies
- [x] Spacing between stuff.
- [x] 50px v 65px line height - increase line height by 30% 

image

### CODE REVIEW

File: www/locales/en.json [hunk 1]
```diff
@@ -195,7 +195,7 @@
     "WRITER": "Writer",
     "SOURCES": {
       "ALL_SOURCES": {
-        "TEXT": "All Sources"
+        "TEXT": "All sources"
```
Comment (tarunsingh5) [inline www/locales/en.json:198]: i feel like the second word being capitalized improved readability

File: www/locales/en.json [hunk 2]
```diff
@@ -221,7 +221,7 @@
     },
     "RAAGS": {
       "ALL_RAAGS": {
-        "TEXT": "All Raags"
+        "TEXT": "All raags"
```
Comment (tarunsingh5) [inline www/locales/en.json:224]: i feel like the second word being capitalized improved readability

File: www/locales/en.json [hunk 3]
```diff
@@ -247,7 +247,7 @@
     },
     "WRITERS": {
       "ALL_WRITERS": {
-        "TEXT": "All Writers"
+        "TEXT": "All writers"
```
Comment (tarunsingh5) [inline www/locales/en.json:250]: i feel like the second word being capitalized improved readability

File: www/locales/en.json (hunk 1 above)
Comment (Gauravjeetsingh) [reply www/locales/en.json]: @tarunsingh5 veerji, yeah I reverted this change back, this was a sideeffect of another text change we did earlier but wasn't pushed.

File: www/locales/en.json (hunk 2 above)
Comment (Gauravjeetsingh) [reply www/locales/en.json]: @tarunsingh5 veerji, yeah I reverted this change back, this was a sideeffect of another text change we did earlier but wasn't pushed.

File: www/locales/en.json (hunk 3 above)
Comment (Gauravjeetsingh) [reply www/locales/en.json]: @tarunsingh5 veerji, yeah I reverted this change back, this was a sideeffect of another text change we did earlier but wasn't pushed.

**Review by inderpreetsingh — APPROVED** [review]

**Pull request merged**